File: VB/DXWebApplication1/Models/EF/DataClassesDataContext.vb
Imports System.Data.Entity
Imports System

Namespace DXWebApplication1.Models.EF

	Partial Public Class DataClassesDataContext
		Inherits DbContext

		Public Sub New()
			MyBase.New("name=DocumentsConnectionString")
		End Sub

		Public Overridable Property Docs() As DbSet(Of Doc)
	End Class
End Namespace

File: CS/DXWebApplication1/Models/EF/DataClassesDataContext.cs
namespace DXWebApplication1.Models.EF {
    using System;
    using System.Data.Entity;

    public partial class DataClassesDataContext : DbContext {
        public DataClassesDataContext()
            : base("name=DocumentsConnectionString") {
        }

        public virtual DbSet<Doc> Docs { get; set; }
    }
}
